### ISSUE TITLE
Credentials not included with cross-origin requests

### ISSUE THREAD

**Issue by uhthomas** [promlabs/promlens-public#15]
Promlens does not work when the remote Prometheus instance is protected by an authenticated proxy (e.g. [OAuth2 Proxy](https://github.com/oauth2-proxy/oauth2-proxy)).

The developer console shows that cookies are not being sent with the request, despite the cookie having `SameSite` set to `None`. Running the following in the developer console from the promlens host works fine.
```js
await fetch('https://prometheus.example.com', { credentials: 'include' })
```

**Comment by juliusv** [comment]
@uhthomas Thanks! And sorry, somehow this slipped under my radar! I'm not sure about the state back then, but I released some newer versions since then (right now, released v1.1.0) and that behavior was touched multiple times since then. At least I think the fetching should now always include `{ credentials: 'include' }` by default. So hopefully this is fixed. I'll close this for now, but let me know if you still run into issues with it.

**Comment by ivasilyev-servicetitan-com** [comment]
@juliusv I have the same issue right now. `_oauth2_proxy` cookie is not sent out. Looking into JS code received from Promlens it seems `credentials` is not always 'include'.
I can see smth like
```
credentials: c.serverSettings.withCredentials ? "include" : "same-origin",
```

and 
```
withCredentials: !1
```

Can you please check if this issue is really solved in v1.1.0?
